addons/mail/static/src/core/public_web/discuss_sidebar.js

```diff
@@ -12,11 +16,22 @@ export const discussSidebarItemsRegistry = registry.category("mail.discuss_sideb
 export class DiscussSidebar extends Component {
     static template = "mail.DiscussSidebar";
     static props = {};
-    static components = {};
+    static components = { Dropdown };
 
     setup() {
         super.setup();
         this.store = useState(useService("mail.store"));
+        this.compactHover = useHover(["compact-btn", "compact-floating*"], () => {
+            this.compactFloating.isOpen = this.compactHover.isHover;
+        });
+        this.compactFloating = useDropdownState();
+    }
+
+    get compactBtnText() {
+        if (this.store.discuss.isSidebarCompact) {
+            return _t("Make panel bigger");
+        }
+        return _t("Make panel smaller");
```

Comment:
Shouldn't we explicitly name this `sidebar` instead of `panel`?  For translation context?
I get that it probably already exists, but I'm not sure that reusing the translation string is a good thing in master.

Reply:
Sidebar is a relatively good name when there was no "inspector", i.e. panel on the right.
Users do not get this component is named sidebar, this is a technical name. They see it as the left panel, and since "panel" is quite clear, I choose this name. The button is on the panel so there's no ambiguity with the right panel.